intl/msg_hash_ja.h

```diff
@@ -2062,9 +2062,9 @@ MSG_HASH(MENU_ENUM_SUBLABEL_BLUETOOTH_ENABLE,
 MSG_HASH(MENU_ENUM_SUBLABEL_CONFIG_SAVE_ON_EXIT,
       "終了時に設定を自動的に保存する。")
 MSG_HASH(MENU_ENUM_SUBLABEL_CONFIGURATION_SETTINGS,
-      "システム設定ファイルのデフォルト設定を変更します。")
+      "全般設定ファイルのデフォルト設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_CONFIGURATIONS_LIST,
-      "システム設定ファイルの作成と管理を行います。")
+      "全般設定ファイルの作成と管理を行います。")
```

Comment:
Perhaps "General" is the best word to clarify what setting "configuration file" refers to. Also, to think separately from the system.